libeos-updater-util/tests/flatpak.c

```diff
@@ -140,13 +143,36 @@ test_compress_uninstall_update_as_uninstall (void)
                    EUU_FLATPAK_REMOTE_REF_ACTION_UNINSTALL);
 }
 
+/* Test that no compresson occurrs if 'uninstall' and 'update' are on
```

Comment:
Nitpick: Typo in ‘occurs’. Just push a fix directly (and grep for other occurrences).

Reply:
One of those ten words I cannot spell to save my life...

Reply:
(Done in #155.)